measure_performance/test_data/new_tests.xml

```diff
@@ -0,0 +1,6 @@
+<AddressCollection>
```

Comment:
We collect test data in in `measure_performance/test_data/labeled.xml`, the "master" testing file. If you could append these addresses to the end of that file it'd help keep the directory clean.